feature-libs/quote/components/header/overview/quote-header-overview.component.ts

```diff
@@ -0,0 +1,257 @@
+   *
+   * @param {string} name - Quote name
+   * @param {string} description - Quote description
+   * @returns {Observable<Card>} - Card content
+   */
+  getQuoteInformation(name?: string, description?: string): Observable<Card> {
+    return combineLatest([
+      this.translationService.translate('quote.header.overview.information'),
+      this.translationService.translate('quote.header.overview.name'),
+      this.translationService.translate('quote.header.overview.description'),
+    ]).pipe(
+      map(([infoTitle, nameTitle, descriptionTitle]) => {
+        return {
+          title: infoTitle,
+          paragraphs: [
+            {
+              title: nameTitle,
+              text: [name ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+            {
+              title: descriptionTitle,
+              text: [description ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+          ],
+        };
+      })
+    );
+  }
+
+  /**
+   * Retrieves the edit card content that represents the edit quote information with its name and description.
+   *
+   * @param {string} name - Quote name
+   * @param {string} description - Quote description
+   * @returns {Observable<EditCard>} - Edit card content
+   */
+  getEditQuoteInformation(name: string, description: string): EditCard {
+    return {
+      name: name,
+      description: description,
+      charactersLimit: QuoteHeaderOverviewComponent.CHARACTERS_LIMIT,
+    };
+  }
+
+  /**
+   * Retrieves the card content that represents the estimated and date information.
+   *
+   * @param {Quote} quote - Quote
+   * @param {any} createdDate - Created date
+   * @returns {Observable<Card>} - Card content
+   */
+  getEstimatedAndDate(
+    quote: Quote,
+    createdDate?: string | null
+  ): Observable<Card> {
+    const totalPrice =
+      this.getTotalPrice(quote) ?? this.getTotalPriceDescription(quote);
+    return combineLatest([
+      this.translationService.translate(
+        'quote.header.overview.estimateAndDate'
+      ),
+      this.translationService.translate('quote.header.overview.estimatedTotal'),
+      this.translationService.translate('quote.header.overview.created'),
+    ]).pipe(
+      map(([firstTitle, secondTitle, thirdTitle]) => {
+        return {
+          title: firstTitle,
+          paragraphs: [
+            {
+              title: secondTitle,
+              text: [totalPrice ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+            {
+              title: thirdTitle,
+              text: [createdDate ?? QuoteHeaderOverviewComponent.NO_DATA],
+            },
+          ],
+        };
+      })
+    );
+  }
+
+  /**
+   * Retrieves the card content that represents the update information.
+   *
+   * @param {string} lastUpdated - last updated time
+   * @param {string} expirationTime - expiration time
+   * @returns {Observable<Card>} - Card content
+   */
+  getUpdate(
```

Comment:
@Larisa-Staroverova 
getUpdate seems to be quite confusing. What about getCardMetadata

Reply:
Done!